comps/vectorstores/langchain/pgvector/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2024 Intel Corporation
```

Comment:
This license header is out of date, please leave it empty (the bot will add it automatically) or add the new one,   
```
Copyright (C) 2024 Intel Corporation   
SPDX-License-Identifier: Apache-2.0   
```

Reply:
Addressed in https://github.com/opea-project/GenAIComps/pull/201